getting-started/cloudflare-workers.md

```diff
@@ -373,4 +373,33 @@ then edit `wrangler.toml`, and add this code after `compatibility_date` line.
 main = "src/index.ts"
 minify = true
 ```
+
 Everything is ready! Now push the code and enjoy it.
+
+## Load env when local development
+
+To configure the environment variables for local development, create the `.dev.vars` file in the root directory of the project.
+Then configure your environment variables as you would with a normal env file.
+
+```
+SECRET_KEY=value
+API_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9
+```
+
+> For more about this section you can find in the Cloudflare documentation:
+> https://developers.cloudflare.com/workers/wrangler/configuration/#secrets
+
+Then we use the `c.env.*` to get the environment variables in our code.  
+**For Cloudflare Workers, environment variables must be obtained via `c`, not via `process.env`.**
+
+```ts
+app.get('/env', (c) => {
+  const SECRET_KEY = c.env?.SECRET_KEY
```

Comment:
`?` is not needed if you specify types correctly:

```ts
const app = new Hono<{
  Bindings: {
    SECRET_KEY: string
  }
}>()
```

We don't recommend to use `?`.

Reply:
Updated. one curiosity as a beginner though, using it here`?` Would there be any problems with it?

Reply:
Thanks. It is just a problem, type-safe or not. If you set `Bindings` you don't need it and it will be type-safed.